source/includes/_templatedata.md

```diff
@@ -0,0 +1,41 @@
+# Template Data
+
+## Briefing Variables
+
+All the data that is available to templates is provided by the BriefingRenderer class.
+
+### `items` > `item` : 
+
+These are all the items in the briefing, with standard item data, plus the following:
+
+* `{{title}}`: The title of the item.
+* `{{link}}`: A link to the content of the item in the Attensa reading view.
+* `{{sourceLink}}`: A link to the content's original source.
+* `{{author}}`: The full name of the author of the item. 
+* `{{defaultDescription}}`: This will display the full text of the item description.
+* `{{cleanDescription}}`: This variable will display the same text as defaultDescription with any HTML stripped off.
+* `{{shortDescription}}`: This variable will display the same text as cleanDescription with a limitation to 200 characters.
+* `{{publishedDate}}`: This will display the item’s published date.
```

Comment:
Currently we don't have it on any of the templates with that data, so far we're just using publishedDate. If theres anyone asking for it, I really don't see a need. Do we have a handle for that?